src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -1,4 +1,400 @@
 privacy_notices:
+### GPP State-by-state ###
+- name: Data sales and sharing
+  notice_key: data_sales_sharing_gpp_us_state
+  id: pri_gpp_1 # TODO: update
+  description: We may transfer or share your personal information to third parties in exchange for monetary or other valuable consideration or for the purposes of [cross-context behavioral targeted advertising]. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Data sales and sharing (GPP, California) # TODO: update?
```

Comment:
yeah i think i'll leave these for now, we can circle back with product before releasing to be sure our copy in general looks good for our notice templates, maybe at that point we can decide to bypass this field altogether....

Reply:
Yep let's verify, current "out of the box" logic doesn't update these after the fact if there's a diff between notices and the templates, because we assume that their edits take precedence.